modules/ob-release-notes-1-5-1.adoc

```diff
@@ -5,19 +5,20 @@
 [id="ob-release-notes-1-5-1_{context}"]
 = Release notes for {builds-shortname} 1.5.1
 
+[role="_abstract"]
 {builds-shortname} 1.5.1 is now available on {ocp-product-title} 4.19.
 
 [id="new-features-1-5-1_{context}"]
 == New features
 
-The following sections highlight what is new in {builds-shortname} 1.5.1.
+The new features in {builds-shortname} 1.5.1 include:
 
 * With this update, {builds-shortname} deployment is FIPS compliant, ensuring that all the cryptographic modules and algorithms meet the security requirements.
 
 [id="fixed-issues-1-5-1_{context}"]
 == Fixed issues
 
-The following list highlights fixed issues in {builds-shortname} 1.5.1.
+The following list highlights fixed issues in {builds-shortname} 1.5.1:
```

Comment:
Just a heads up about "Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​"." from https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc.